expyfun/_input_controllers.py

```diff
@@ -84,6 +99,17 @@ def _on_pyglet_keypress(self, symbol, modifiers, emulated=False):
             this_key = this_key.lstrip('_').lstrip('NUM_')
         self._keyboard_buffer.append((this_key, key_time))
 
+    def _on_pyglet_keyrelease(self, symbol, modifiers, emulated=False):
```

Comment:
as far as I can tell, the definition of `_retrieve_keyboard_releases` and `_on_pyglet_keyrelease` are identical to the corresponding keypress methods.  That would mean that what you're actually returning is key press times, not key release times.  Am I missing something?